Makefile

```diff
@@ -124,8 +124,16 @@ E2E_REGISTRY_NAME := docker-registry
 E2E_REGISTRY_NAMESPACE := operator-controller-e2e
 
 export REG_PKG_NAME := registry-operator
-export REGISTRY_ROOT := $(E2E_REGISTRY_NAME).$(E2E_REGISTRY_NAMESPACE).svc:5000
-export CATALOG_IMG := $(REGISTRY_ROOT)/e2e/test-catalog:e2e
+export LOCAL_REGISTRY_HOST := $(E2E_REGISTRY_NAME).$(E2E_REGISTRY_NAMESPACE).svc:5000
+export CLUSTER_REGISTRY_HOST := localhost:30000
```

Comment:
Not sure what others think, but to me it is more logical to flip these. LOCAL for `localhost`, CLUSTER for the service-based name.

Reply:
I wouldn't call `localhost` a host, this is the `ENDPOINT` into the the cluster. It's really the `external` callable address. Spitballing, because names are hard, but maybe `CLUSTER_REGISTRY_INGRESS`? 

I agree the `svc:5000` address should be called `CLUSTER`.

Reply:
Do we need to address this before merging? Looks like we should...?